installer.py

```diff
@@ -19,27 +17,11 @@ def ensure_product_choice(app=None):
     logging.debug('- config.FLPRODUCTi')
     logging.debug('- config.VERBUM_PATH')
 
-    if not config.FLPRODUCT:
```

Comment:
The reason for having the `if not config.VARIABLE` statements at each install step was to be able to skip the step if the variable was already set. Without them each step gets processed in full, even if the user has already set the variable, either in the config file or with env variables.

Reply:
I believe I commented those out from an earlier revision, before I had set up proper threading in the TUI. Those should be easy to restore.